### PR TITLE
CVS-80230 mapping layout fix

### PR DESCRIPTION
**Pull request by ficol**

### CODE REVIEW

**Review by atobiszei — APPROVED** [review]

File: src/test/modelinstance_test.cpp [hunk 1]
```diff
@@ -642,8 +657,8 @@ TEST_F(TestReloadModelWithMapping, SuccessfulReload) {
     shapeMap["input"] = inputShape;
     config.setShapes(shapeMap);
 
-    layouts["input"] = "LAYOUT_INPUT";
-    layouts["output"] = "LAYOUT_OUTPUT";
+    layouts["input"] = "NC";
```
Comment (dkalinowski) [inline src/test/modelinstance_test.cpp:660]: Why this change?

Comment (ficol) [reply src/test/modelinstance_test.cpp:660]: Because openvino doesn't accept this when creating ov::Layout here:
https://github.com/openvinotoolkit/model_server/blob/ad24df4970a131235f2421ef44dc71ad44df0697/src/modelinstance.cpp#L208-L209

Comment (dkalinowski) [reply src/test/modelinstance_test.cpp:660]: How comes this test was passing before?

Comment (ficol) [reply src/test/modelinstance_test.cpp:660]: Because before change, code fell into last else branch instead of into:
else if (config.getLayouts().count(mapped_name) > 0)

**Review by dkalinowski — APPROVED** [review]
